### PR TITLE
Allow the user to setup backend DB url in ert3

### LINKED ISSUE #2673: Allow configuration of database when using ert3
The entry point of starting the storage server through ert3 currently does not allow the user to configure the database:
https://github.com/equinor/ert/blob/a495b05edfb0798b928bbe0b37858368e23af6e7/ert3/console/_console.py#L307-L309

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
**Issue**
Resolves #2673 

**Approach**
Add `database_url` argument when using `ert3 service storage start`

## Pre review checklist

- [x] Added appropriate labels

Adding labels helps the maintainers when writing release notes, see sections and the
corresponding labels here: https://github.com/equinor/ert/blob/main/.github/release.yml

### CODE REVIEW

File: ert3/console/_console.py [hunk 1]
```diff
@@ -113,6 +113,20 @@ def _build_clean_argparser(subparsers: Any) -> None:
     )
 
 
+def _build_start_storage_argparser(subparsers: Any) -> None:
+    start_storage_parser = subparsers.add_parser(
```
Comment (xjules) [inline ert3/console/_console.py:117]: Is `start_storage_parser` variable used somewhere? If not I wouldn't create it ...

**Comment by lars-petter-hauge** [comment]
Tests fails with:

```
23 File "/opt/hostedtoolcache/Python/3.6.15/x64/lib/python3.6/site-packages/uvicorn/server.py", line 67, in run
24 return asyncio.run(self.serve(sockets=sockets))
25 AttributeError: module 'asyncio' has no attribute 'run'
26 ERROR: Ert storage not found!
27 Error: Process completed with exit code 1.
```

Looks like incompatibility between `uvicorn` and `asyncio`, restarting jobs now (since that is what I have time for). Will check versions if it continues to fail

**Comment by berland** [comment]
Rebase, and uvicorn should be pinned for you.

**Comment by lars-petter-hauge** [comment]
> Rebase, and uvicorn should be pinned for you.

Thank you, done!

**Review by sondreso — APPROVED** [review]
LGTM!

**Pull request merged**